packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -320,6 +320,41 @@ test('Strings', async t => {
   });
 });
 
+test('JSON blobs', async t => {
+  const paramManager = makeParamManagerBuilder()
+    .addNat('Acres', 50n)
+    .addJson('Blob', '{ "author": "Crockford" }')
+    .build();
+  t.is(paramManager.getJson('Blob'), '{ "author": "Crockford" }');
+
+  await paramManager.updateParams({
+    Blob: '{ "authors": "Crockford,Morningstar" }',
+  });
+  t.is(paramManager.getBlob(), '{ "authors": "Crockford,Morningstar" }');
+  await t.throwsAsync(
+    () =>
+      paramManager.updateParams({
+        Blob: 300000000,
+      }),
+    {
+      message: 'Expected a json string. "300000000" is not valid json.',
+    },
+  );
+  await t.throwsAsync(
+    () =>
+      paramManager.updateParams({
+        Blob: 'author: bob',
```

Comment:
this seems quite similar to the test above.

some tests for common manual editing:
- valid JSON with trailing whitespace passes
- valid JSON with trailing semicolon fails

Reply:
done